daemons/controld/controld_remote_ra.c

```diff
@@ -192,6 +192,9 @@ remote_node_up(const char *node_name)
     if (controld_shutdown_lock_enabled) {
         section = controld_section_all_unlocked;
     }
+    /* Purge node from attrd's memory */
+    update_attrd_remote_node_removed(node_name, NULL);
+
     controld_delete_node_state(node_name, section, call_opt);
 
     /* Clear node's probed attribute */
```

Comment:
BTW as a side note, this update_attrd() for CRM_OP_PROBED seems no longer necessary, but as a matter of fact attrd still relies on this to create a membership cache entry for the node as a *remote* node.